src/scss/mixins/_common.scss

```diff
@@ -98,15 +95,38 @@
 	&--wide {
 		@include oFormsFullWidth;
 	}
+}
+
+@mixin oFormsRadioCheckboxGroup($selectors: (radio: '.o-forms__radio', checkbox: '.o-forms__checkbox', checkbox-toggle: '.o-forms__checkbox-toggle__checkbox', label: '.o-forms__label')) {
+	$checkbox-toggle: unquote(map-get($selectors, checkbox-toggle));
+	$checkbox: unquote(map-get($selectors, checkbox));
+	$radio: unquote(map-get($selectors, radio));
+	$label: unquote(map-get($selectors, label));
 
 	&__checkboxes,
 	&__radios {
 		margin: 0 0 #{-$_o-forms-spacing};
 		#{$radio} + #{$label},
+		#{$checkbox-toggle} + #{$label},
 		#{$checkbox} + #{$label} {
 			width: 100%; // We do not use display:block to maintain consistent margins (not collapsed).
 			box-sizing: border-box;
+			margin: $_o-forms-checkbox-legend-spacing 15px $_o-forms-spacing 0; // margins do not collapse, making the space between checkbox/radio inputs a little larger
+			&:last-of-type {
+				margin-right: 0;
+			}
+		}
+
+		$align-right: '--right';
```

Comment:
doesn't this technically align left ...?

Reply:
@gvonkoss 🤔 it aligns the checkbox input to the right of the label, as the modifier is applied to the input I think this makes sense?

Reply:
oh I _see_.
I thought we were referring to the whole of that section being aligned to the bottom of the input 🤦‍♀️ 
As in the `show password  []` is the suffix that we'd be aligning, not the actual checkbox in relation to the text. 

Would that also be an example of 'information' you can affix as a suffix? 😬

Reply:
This is no longer in a suffix, it's just a checkbox (aligned right to the label) under an input
<img width="431" alt="35163073-66cb0a4a-fd3d-11e7-960c-6884cc7d239c" src="https://user-images.githubusercontent.com/10405691/35220899-138a40be-ff70-11e7-9479-ecc2e7f55a1c.png">
Does that make sense? We might want to change the markup/demos to be clearer.